str/runners/str_iterative_eh_runner.py

```diff
@@ -3,22 +3,24 @@
 
 """
 This script uses ExpansionHunterv5 to call STRs on WGS cram files.
-Required input: --variant-catalog (file path to variant catalog), --dataset, and internal sample IDs
+Required input: --variant-catalog (file path to variant catalog, can be sharded or unsharded), --dataset, and sample mapping file [CSV] (CPG sample id (first column) and sex (second column))
+EH will run on every sample listed inthe sample mapping file.
+
 For example:
-analysis-runner --access-level test --dataset tob-wgs --description 'tester' --output-dir 'tester' str_iterative_eh_runner.py --variant-catalog=gs://cpg-tob-wgs-test/hoptan-str/Illuminavariant_catalog.json --dataset=tob-wgs CPG26 CPG18
+analysis-runner --access-level test --dataset tob-wgs --description 'tester' --output-dir 'tester' str_iterative_eh_runner.py --variant-catalog=gs://cpg-tob-wgs-test/hoptan-str/Illuminavariant_catalog.json --dataset=tob-wgs --sample-id-file=gs://...
 
 Required packages: sample-metadata, hail, click, os
```

Comment:
Could you add a `requirements.txt` file in the top folder of the project, with sample-metadata, hail, and click (ideally with pinned versions e.g. `click>=8`, `metamist==6.5.0` as appropriate)

Can you also remove references to `sample-metadata` and replace with `metamist`